### ISSUE TITLE
Git clone node with shallow

### ISSUE THREAD

**Issue by joshughes** [vercel/pkg-fetch#46]
Cloning the entire nodejs repo on each build is slow. 

```
--branch TAG  --depth 1
```

Will be more efficient.

**Comment by biancode** [comment]
like:

```
async function gitClone (nodeVersion) {
  log.info('Cloning Node.js repository from GitHub...');
  const args = [ 'clone', `-b ${nodeVersion}`, '--depth 1', '--single-branch', '--bare', '--progress', nodeRepo, 'node/.git' ];
  const promise = spawn('git', args, { cwd: buildPath });
  progress(promise, thresholds('clone'));
  await promise;
}
```

**Comment by hipstersmoothie** [comment]
@leerob This was fixed In #84, this issue can be closed.
